src/package/packageVersionCreate.ts

```diff
@@ -374,7 +374,7 @@ export class PackageVersionCreate {
       packageDescriptorJson = copyDescriptorProperties(packageDescriptorJson, definitionFileJson);
     }
 
```

Comment:
A corresponding schema change to sfdx-project.json is required for these changes.

Reply:
schema change has been merged: https://github.com/forcedotcom/schemas/commit/a1db98d229b8a6fd292f29a78acf7947027500f2